src/app/components/pages/blocks/blocks.component.ts

```diff
@@ -85,7 +90,12 @@ export class BlocksComponent implements OnInit, OnDestroy {
     private api: ApiService,
     private explorer: ExplorerService,
     private route: ActivatedRoute,
-  ) { }
+  ) {
+    // Each time the node URL is changed, check if a local node is being used.
+    this.nodeUrlSubscription = this.api.localNodeUrl.subscribe(response => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
    this.nodeUrlSubscription = this.api.localNodeUrl.subscribe((response) => {
```